src/renderer/modals/login/loginModal.tsx

```diff
@@ -87,82 +91,125 @@ export default class LoginModal extends React.PureComponent<Props, State> {
     }
 
     render() {
+        const {intl} = this.props;
+
         let theServer = '';
         if (!(this.state.request && this.state.authInfo)) {
             theServer = '';
         } else if (this.state.authInfo.isProxy) {
-            theServer = `The proxy ${this.state.authInfo.host}:${this.state.authInfo.port}`;
+            theServer = intl.formatMessage({
```

Comment:
This also has the concatenation issue. In this case, I think it would be easiest to just include the "requires a username and password" in each of these strings.

We could also use FormattedMessage here then instead of injecting the intl context